webapp/sources/rudder/rudder-web/src/main/javascript/rudder/tree.js

```diff
@@ -35,6 +35,28 @@
 *************************************************************************************
 */
 
+
+  var allEvents = [];
+
+  // Since #24062, we inlined on click events from our lift generated nodes,
+  // there nodes are transformed into a tree using jstree
+  // jstree does not keep the event that were bound to the previous nodes
+  // We need to reattach the events here
+  function attachInlinedEvents( id ) {
+    var child = $(id).find('[id^="lift-event-js-"]'); // all nested child elements with a 'lift-event-js-...' id
+    child.each(function () {
+      var nodeId = this.id;
+      var nodes = allEvents.find(function (x) { return x[0].id === nodeId });
+      if (nodes && nodes[1]) {
+        var event = nodes[1];
+        if (event) {
+          $(this).off("click");
```

Comment:
added this to prevent adding again another click event, another cause of this issue